### PR TITLE
Actions

### PR DESCRIPTION
**Pull request by Fettpet**
Add github actions
* change the datatypes of mysql, see issue #22. This could be local a problem

### CODE REVIEW

File: src/src.qbs [hunk 1]
```diff
@@ -6,6 +6,12 @@ DynamicLibrary {
     cpp.includePaths: ["./"]
     cpp.defines: ["LOG_PREFIX=\"[QSqlMigrator]\"",
                   "_BUILDING_QSQLMIGRATOR_DLL"]
+    Properties {
+        condition: qbs.toolchain.contains("clang")
+        cpp.commonCompilerFlags: "-Wno-deprecated-copy"
```
Comment (arBmind) [inline src/src.qbs:14]: I guess this should be exported here, so all Libs and Applications that depend on QSqlMigrator automatically inherit this flag?

**Comment by arBmind** [comment]
Good work so far. 👍 

It seems on Windows the tests are not actually run. No console Output of the execution was recorded.
Installing the Qt Plugins seems to miss out on PostgreSql. Maybe a library path is missing?

These messages also seem strange, but are also present on Ubuntu builds:
```
NOTICE:  database "qdb_migrator_test_db" does not exist, skipping
```

File: tests/api/api.qbs [hunk 2]
```diff
@@ -1,12 +1,18 @@
+import qbs
+import qbs.FileInfo
```
Comment (arBmind) [inline tests/api/api.qbs]: These imports seem to be unused.

**Comment by Fettpet** [comment]
> It seems on Windows the tests are not actually run. No console Output of the execution was recorded.

Because of an unknown reason, the windows tests don't output anything. They are running, since if an error occurs, they exit with an exit code != 0. A workaround is to write the output into a file and output these file. 

>  Installing the Qt Plugins seems to miss out on PostgreSql. Maybe a library path is missing?

The qt plugin for PostgreSQL is installed within the action jurplel/install-qt-action@v2. It doesn't install the mysql plugin.

> These messages also seem strange, but are also present on Ubuntu builds:
> NOTICE:  database "qdb_migrator_test_db" does not exist, skipping

I dont see the message on ubuntu builds.  I think it came from  PostgresqlTest::testTransaction. [See](https://github.com/Fettpet/QSqlMigrator/runs/3650579717?check_suite_focus=true#step:11:136)

**Pull request closed without merging**